### ISSUE TITLE
Failure during the execution of install_openstack.sh

### ISSUE THREAD

**Issue by Aledangelo** [dessertlab/OpenStack-Fault-Injection-Environment#1]
I think that the mirror used in this script is no more available. I tried on my own instance of Centos 7.9.2009, alternatively I also tried with VMWare image downloaded from the link present in `INSTALL.md` file.

I also tried changing the mirrors link in the script installation, but it doesn't work either.

This is the last part of the output
```
...

One of the configured repositories failed (added from: http://vault.centos.org/7.9.2009/cloud/x86_64/openstack-pike),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=vault.centos.org_7.9.2009_cloud_x86_64_openstack-pike ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable vault.centos.org_7.9.2009_cloud_x86_64_openstack-pike
        or
            subscription-manager repos --disable=vault.centos.org_7.9.2009_cloud_x86_64_openstack-pike

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=vault.centos.org_7.9.2009_cloud_x86_64_openstack-pike.skip_if_unavailable=true


failure: repodata/repomd.xml from vault.centos.org_7.9.2009_cloud_x86_64_openstack-pike: [Errno 256] No more mirrors to try.
http://vault.centos.org/7.9.2009/cloud/x86_64/openstack-pike/repodata/repomd.xml: [Errno 14] HTTPS Error 404 - Not Found
2023-03-09 11:04:49.488 Disable firewalld, NetworkManager and enable network module
Removed symlink /etc/systemd/system/dbus-org.fedoraproject.FirewallD1.service.
Removed symlink /etc/systemd/system/multi-user.target.wants/firewalld.service.
Removed symlink /etc/systemd/system/dbus-org.freedesktop.nm-dispatcher.service.
Removed symlink /etc/systemd/system/dbus-org.freedesktop.NetworkManager.service.
Removed symlink /etc/systemd/system/multi-user.target.wants/NetworkManager.service.
Removed symlink /etc/systemd/system/network-online.target.wants/NetworkManager-wait-online.service.
network.service is not a native service, redirecting to /sbin/chkconfig.
Executing /sbin/chkconfig network on
2023-03-09 11:04:50.866 Prepare packstack configuration file
2023-03-09 11:04:50.876 Local IP: 10.0.2.15
2023-03-09 11:04:50.879 Start packstack deployment
./src/install/install_openstack.sh: line 99: packstack: command not found
2023-03-09 11:04:50.882 Check if OpenStack env is deployed...
  File "/home/osboxes/OpenStack-Fault-Injection-Environment/src/install/check_openstack.py", line 9, in <module>
    from keystoneauth1 import identity
2023-03-09 11:04:50.895 WARNING OpenStack is not properly installed!
```

**Comment by piliguori** [comment]
It is a problem with the OpenStack Pike version, which is no anymore available on http://vault.centos.org/

I modified the repo on the install_openstack.sh script. This should fix the issue. Let me know if it works. Thanks

**Comment by Aledangelo** [comment]
Yes, on VMWare image it works fine!

```
...

 * The installation log file is available at: /var/tmp/packstack/20230310-040459-KMgp_v/openstack-setup.log
 * The generated manifests are available at: /var/tmp/packstack/20230310-040459-KMgp_v/manifests
 * Note temporary directory /var/tmp/packstack/d9d590b51786411694a54270cb675185 on host 10.0.2.15 was not deleted for debugging purposes.
2023-03-10 04:18:27.506 Check if OpenStack env is deployed...
2023-03-10 04:18:28,785 OpenStack is properly installed!
```
